docs/source/experiment_tracking.md

```diff
@@ -48,7 +48,7 @@ There are three steps to enable experiment tracking features with Kedro-Viz. We
 - [Modify your nodes and pipelines to output those metrics](#modify-your-nodes-and-pipelines-to-log-metrics)
 
 ### Install Kedro and Kedro-Viz
-To use this tutorial code, you must already have [installed Kedro](https://docs.kedro.org/en/stable/get_started/install.html) and [Kedro-Viz](./kedro-viz_visualisation.md). You can confirm the versions you have installed by running `kedro info`
+To use this tutorial code, you must already have {doc}`installed Kedro<kedro:get_started/install>` and [Kedro-Viz](./kedro-viz_visualisation.md). You can confirm the versions you have installed by running `kedro info`
```

Comment:
For the future: maybe Kedro Viz needs its own installation page, rather than the current link?

Reply:
Definitely! I was sick of making the same update. There's quite a lot needs to change though because of the add-ons and starter changes, so I haven't done it right now.